engines/grim/emi/lua_v2.cpp

```diff
@@ -522,8 +527,10 @@ void Lua_V2::ThumbnailFromFile() {
 
 void Lua_V2::GetMemoryCardId() {
 	// 0 - No mem card
-	lua_pushnumber(0);
-	warning("GetMemoryCardId: Currently just returning 0");
+	// 1 - Not formatted
+	// 2 - Not enough space
+	// 3 - Error occured
```

Comment:
No other numbers seem to be specifically checked for.  I'm not sure what a valid id would look like on a real console but returning "4" is apparently enough to let saving work.